### PR TITLE
Python parser

### PR DESCRIPTION
**Pull request by guihomework**
Following the discussions in https://github.com/ros-planning/srdfdom/issues/8 this PR adds support for a python parser in srdfdom based on the work done in urdf_parser_py and on its reflection library

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -1,11 +1,16 @@
 <package>
   <name>srdfdom</name>
-  <version>0.2.7</version>
+  <version>0.2.8</version>
   <description>Parser for Semantic Robot Description Format (SRDF).</description>
   <author email="isucan@willowgarage.edu">Ioan Sucan</author>
+  <author email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</author>
+  <author>Wim Meeussen</author>
+  <author>John Hsu</author>
```
Comment (davetcoleman) [inline package.xml]: Why are these guys being added as authors? They have never been involved with SRDF afaik

**Comment by davetcoleman** [comment]
This is an impressive PR - thanks for creating tests, a demo script, improving documentation, etc. I'm not really a python programmer so can't speak to that part's quality, but from my review everything looks good, except:
- Could you document the display_srdf script in the README?
- Could you document how to use/run the test? This might be obvious and a standard ROS thing, but I always forget

File: package.xml (hunk 1 above)
Comment (guihomework) [inline package.xml]: It comes from the urdfdom from which I copied the parser and adapted it to srdf (https://raw.githubusercontent.com/ros-gbp/urdfdom-release/debian/indigo/trusty/urdfdom/package.xml) 
But if looking in details, the people added in the README.md are the correct ones I derived the work from. 
I don't know how to acknowledge these people.

**Comment by guihomework** [comment]
while adding more tests (the same as in the cpp parser) I found some tiny bugs in the python parser. Currently fixing them.
Why is travis building for precise ? This is indigo-devel branch and I don't remember indigo was supported on precise.

**Comment by davetcoleman** [comment]
travis does not support anything newer, unfortunately. we've turned off travis support in most of the moveit repos already, actually. i just turned it off for this one, too.

File: package.xml (hunk 1 above)
Comment (davetcoleman) [inline package.xml]: I think what you have in the README is sufficient, if we made everyone authors of new packages based on references to old code, we would have very long author lists. Generally, only major contributors to a package who have specifically worked on that one claim authorship. I think you should remove the other ones.

**Comment by guihomework** [comment]
I did the suggested changes. I also noticed the cpp_parser test is not started with rostest but I will create a new PR for that after this one is merged (to separate issues)

**Comment by davetcoleman** [comment]
Great, thanks for addressing my feedback!

**Comment by ugocupcic** [comment]
@davetcoleman Could you tell me if you're planning to do a new release in the near future? We'd like to use @guihomework new feature :).

Cheers,

**Comment by davetcoleman** [comment]
Just released, again sorry for delay

**Comment by ugocupcic** [comment]
@davetcoleman thanks !

**Pull request merged**